### PR TITLE
First task: adding myself as a member

### PR DESCRIPTION
**Pull request by lilcser**

### CODE REVIEW

**Comment by chrisjluc** [comment]
Looks like you duplicated a bunch of people, if you run the website locally you'll see what I mean!

**Comment by hkjayakumar** [comment]
@lilcser Can you update the latest version of dev on your local and resolve the conflicts in your branch?
It might be easier to delete and re-create the branch, since a couple of people have added their names to dev.

File: src/app/team-page/team-page.component.html [hunk 1]
```diff
@@ -118,11 +118,16 @@
         <p class="name">Ioana Crant</p>
         <p class="title">project developer</p>
       </div>
-      <div class="col-md-3 col-sm-6">
+       <div class="col-md-3 col-sm-6">
```
Comment (hkjayakumar) [inline src/app/team-page/team-page.component.html]: Can you remove the space before `<div`

File: src/app/team-page/team-page.component.html [hunk 2]
```diff
@@ -118,11 +118,16 @@
...
         <img src="https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fprofiles%2Fpratyush.jpg?alt=media&token=06ea4236-3eee-4508-8b77-4feacaea33c5">
         <p class="name">Pratyush Pal</p>
         <p class="title">project developer</p>
       </div>
+      <div>
```
Comment (hkjayakumar) [inline src/app/team-page/team-page.component.html]: Can you make this `<div class="col-md-3 col-sm-6">`

**Pull request merged**